CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -6,6 +6,10 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.8 (TBD)
 
+### Go
+- Require [Go 1.19+](https://github.com/etcd-io/etcd/pull/15337).
+- Compile with [Go 1.19+](https://go.dev/doc/devel/release#go1.19)
+
```

Comment:
This isn't an issue, but could you please put these at the end, so as to be consistent with previous go bumping e.g in 3.5.7?

Reply:
Absolutely, done! 👍🏻